classes/Config.js

```diff
@@ -18,8 +19,12 @@ class Config {
     try {
     	const endpoint = `https://api.github.com/repos/${GITHUB_ORG_NAME}/${this.siteName}/contents/_config.yml`
 
-	    const resp = await axios.get(endpoint, {
-	      validateStatus: validateStatus,
+			const params = {
+        validateStatus: validateStatus,
+        "branch": this.branchRef,
```

Comment:
this.branchRef is not defined. We can just set this to be BRANCH_REF

Reply:
There’s another param with branch on line 52

Reply:
Removed all mentions of `this.branchRef` and use `BRANCH_REF` instead